src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -67,6 +67,14 @@ public boolean hasPerson(Person person) {
         return persons.contains(person);
     }
 
+    /**
+     * Returns true if a person with the same policy number as {@code person} in the address book.
+     */
+    public boolean hasPolicyNumber(Person person) {
```

Comment:
I think the naming of this function can be more clear like `hasSamePolicyNumber` because `hasPolicyNumber` sounds like it is checking if the policy number exist.

```suggestion
    public boolean hasSamePolicyNumber(Person person) {
```